index.js

```diff
@@ -32,15 +32,23 @@ module.exports = function (opts = {}) {
     }
 
     app.authboot = {};
-    const lookup = app.authboot.lookup = (lookupOpt || function ({ name, password }, callback) {
-      const hash = users.get(name);
-      if (!hash) {
-        debug(`unknown username ${name}`);
-        return callback(null, false);
-      }
-      bcrypt.compare(password, hash, callback);
-    });
+    let lookup = lookupOpt;
+
+    // If we have nothing to authenticate against then we will noop
```

Comment:
That seems wrong.  That feels like a configuration bug. Why did you include this package and set it up if you aren't going to actually authenticate?

If this is for local testing, I could see supporting a configuration that is explicitly "don't auth", or you just conditionally supply your own noop configuration.

Either way, this should definitely be **very** _well_ documented in the readme.  Either with the gotcha about it, or how to temporarily opt-out.

Reply:
Ya, maybe better to just _always_ 401/403 if no auth is properly configured. With perhaps a config setting to force open auth.